tensorboard/webapp/hparams/_types.ts

```diff
@@ -27,8 +29,39 @@ export {
   HparamSpec,
   HparamsValueType,
   MetricSpec,
+  Domain,
+  HparamValue,
+  RunToHparamsAndMetrics,
 } from '../runs/data_source/runs_data_source_types';
 
+export {
+  HparamSpec as BackendHparamSpec,
+  DiscreteDomainHparamSpec,
+  BackendHparamsExperimentResponse,
+  BackendListSessionGroupResponse,
+  BackendListSessionGroupRequest,
+  BackendHparamsValueType,
+  RunStatus,
+} from '../runs/data_source/runs_backend_types';
+
+export interface HparamsAndMetricsValues {
+  hparams: HparamValue[];
+  metrics: MetricValue[];
+}
+
+export interface HparamsAndMetricsSpecs {
+  hparams: HparamSpec[];
+  metrics: MetricSpec[];
+}
+
+export type ExperimentToHparamsMetrics = Record<
```

Comment:
I am assuming these types are being used in future PRs? I think I would prefer to add them when they are first used.

Reply:
Yes, these will get used in the effects file. Would you rather I wait to add them?